examples/experimental/gltf-with-raw-webgl/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <title>Raw WebGL (partial) GLTF loader</title>
+
+        <script type="text/javascript" src="lib/gl-matrix.js"></script>
+        <script type="text/javascript" src="lib/loaders.gl.webpack/lib/loaders.gl.js"></script>
```

Comment:
At least in this official example, I'd prefer if we loaded the pre-built loaders.gl libraries from CDN rather than made a custom bundle. We can do that separately as a second pass.

Reply:
OK